torchvision/models/alexnet_onlyFullyConnected.py

```diff
@@ -0,0 +1,48 @@
+import torch
+import torch.nn as nn
+from .utils import load_state_dict_from_url
+
+
+__all__ = ['AlexNet', 'alexnet']
+
+
+model_urls = {
+    'alexnet': 'https://download.pytorch.org/models/alexnet-owt-4df8aa71.pth',
```

Comment:
These weights are only valid for the `AlexNet` structure as it is right now. Thus, within this `state_dict` you would also have weights for the convolutional layers which will result in an error if you try to load it. Furthermore, since you changed the dimensions of the input layer, the weights cannot be reused.